database/dataGenerator.js

```diff
@@ -10,26 +10,28 @@ for (let i = 1; i < 100; i += 1) {
     loc_name: faker.address.city(),
   });
   // generate random number of reviews and user for each location
-  for (let x = 1; x < Math.floor(Math.random() * Math.floor(15)); x += 1) {
+  for (let x = 1; x < Math.floor(Math.random() * Math.floor(15) + 5); x += 1) {
     // save each user to users table in user_reviews database
```

Comment:
For simplicity, consider using faker to generate random numbers.

Example code that generates a random number between 10 and 500:
`faker.random.number({min: 10, max: 500})`